src/cat.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const CID = require('cids')
+const { Buffer } = require('buffer')
+const configure = require('./lib/configure')
+const toIterable = require('./lib/stream-to-iterable')
+
+module.exports = configure(({ ky }) => {
+  return (path, options) => (async function * () {
+    options = options || {}
+
+    const searchParams = new URLSearchParams(options.searchParams)
+
+    if (typeof path === 'string') {
+      searchParams.set('arg', path)
+    } else {
+      searchParams.set('arg', new CID(path).toString())
+    }
+
+    if (options.offset) searchParams.set('offset', options.offset)
+    if (options.length) searchParams.set('length', options.length)
+
+    const res = await ky.get('cat', {
+      timeout: options.timeout,
+      signal: options.signal,
+      headers: options.headers,
```

Comment:
docs only has offset and length for options https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/FILES.md#cat.

Reply:
These are not options for the core interface but they will be available to all methods in the HTTP client. They need to be documented when they are supported globally. I’ll create an issue to track this and ensure it gets done.